scripts/changelog.js

```diff
@@ -237,7 +238,7 @@ class RepositoryFetcher {
 			}
 		}`;
 		const data = await this.fetch(tagQuery, {tag});
-		return data.repository.ref.tag.commit;
+		return data.repository.ref.tag.commit || data.repository.ref.tag;
```

Comment:
Looks like the v3.0.0 tag for `thelounge` wasn't attached to a commit like the other tags are, but on `thelounge.github.io`, it is. Beats me.